React/Card/info-card/README.md

```diff
@@ -0,0 +1,41 @@
+# Rotating Image / Card
+Rotating Image / Card Component
+
```

Comment:
I feel that you forgot to change the title.

Reply:
Sorry I missed it while updating the documentation.